### PR TITLE
6

### PR DESCRIPTION
**Pull request by SidneyPlusnet**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -109,4 +109,71 @@ expect(reviews).toBeSortedBy('created_at',  {
...
+                        votes: expect.any(Number),
+                        created_at : expect.any(String),
+                        author: expect.any(String),
+                        body: expect.any(String),
+                        review_id : expect.any(Number),
+                    })
+                )
+            })
+        })
+    })
```
Comment (johnhutcheon) [inline __tests__/app.test.js:145]: Could you just add one more test here for a status 200, valid ID but has no comments? This should respond with an empty array of comments. 👍

**Review by johnhutcheon — CHANGES_REQUESTED** [review]
Hi Sidney, nice work, looking good! Could you just resubmit once you've added the extra test I've commented about please? Thank you! 🙂
